src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/CSharp/DefaultCSharpCodeActionResolver.cs

```diff
@@ -95,7 +99,7 @@ internal class DefaultCSharpCodeActionResolver : CSharpCodeActionResolver
                 return codeAction;
             }
 
-            if (resolvedCodeAction.Edit.DocumentChanges.Count() != 1)
+            if (resolvedCodeAction.Edit.DocumentChanges.Value.Count() != 1)
```

Comment:
~What is the type of DocumentChanges? Seems like VS types usually use arrays, not IEnumerable, so can this be changed to `.Length` or `.Count` now?~ Nope! SumType sucks!!!

Reply:
Yeah, the implementation of SumType itself and of this as a nested SumType makes this quite awkward. What's worse is extension methods are our only way of dealing with this because SumType is sealed. If we find that working with this continues to be annoying after I finish these conversions is might be worthwhile to create a class which is CONVERTABLE to SumType and use that instead. But that's probably not worth it unless it causes someone other than me pain.

Reply:
Honestly, not that I can think of better alternatives, but the fact that these methods look like Linq methods is what really got me worried. If the names were different I probably wouldn't have even commented 😛